app/stacks/uk-south/back-office/app-service-config.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_app_configuration" "back_office" {
+  count = var.is_dr_deployment ? 1 : 0
+
+  name                = "pins-asc-back-office-${local.resource_suffix}"
+  resource_group_name = azurerm_resource_group.back_office_stack.name
+  location            = module.azure_region_uks.location
+  sku                 = "standard"
+
+  tags = local.tags
+}
```

Comment:
We might want to check our DR strategy for this; but this is OK for now (and presumably follows appeals). Given the config will live in terraform, it is straight forward to deploy this to uk-south when required.